### PR TITLE
AQCU-1035 CORR- Always show the threshold lane

### PR DESCRIPTION
**Pull request by mhines-usgs**
-creating an empty threshold row with the start/end dates from reportMetadata and a null label; if there are threshold data these are overwritten with actual data, if not, an empty row appears.

### CODE REVIEW

**Comment by mhines-usgs** [comment]
image

**Review by zmoore-usgs — CHANGES_REQUESTED** [review]
These changes seem to work but I think there is a somewhat cleaner way to get the lane to always show up. If you look in the render file the call to `createPlotLanes` takes arguments of `optionalData` and `requiredData`. I think if you move the threshold stuff from optional to required it should make the lane always show. You might have to adjust the math for `rectHeight` in `createPlotLanes` though because I think it's based on the number of required/optional lanes.

**Comment by mhines-usgs** [comment]
Thanks @zmoore-usgs -- checking that out. I think I've got it working how you suggest with the few examples from the JIRA now, but the height is still a little funky. Will keep looking.

**Comment by zmoore-usgs** [comment]
Woah, my comment ended up in some weird place where it doesn't show up in the files changed view... 😕  I'll try again

File: R/correctionsataglance-render.R [hunk 1]
```diff
@@ -41,7 +41,8 @@ correctionsataglanceReport <- function(reportObject) {
   dataLanes <- allLaneData[['dataLanes']]
   processOrderLabelYPos <- mean(c(dataLanes[['preData']][["laneNameYPos"]],
                                   dataLanes[['normalData']][["laneNameYPos"]],
-                                  dataLanes[['postData']][["laneNameYPos"]]))
+                                  dataLanes[['postData']][["laneNameYPos"]],
+                                  dataLanes[['thresholdData']][["laneNameYPos"]]))
```
Comment (zmoore-usgs) [inline R/correctionsataglance-render.R]: This part of the code is adjusting the y position of the "processing order" label to be in the middle of the processing order lanes. I'm not sure if threshold data is supposed to go under the "Processing Order" label or not? I don't think it was under it before when the lane show up, though I'm not totally sure.

**Review by zmoore-usgs — APPROVED** [review]
Changes look good, just a question about the processing order label. If threshold data is supposed to be under it then I think this looks good!

**Pull request merged**